ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionRequestForm.tsx

```diff
@@ -32,6 +32,7 @@ function getDefaultValues(cves: string[], scopeContext: ScopeContext): Exception
 }
 export type ExceptionRequestFormProps = {
     cves: CveSelectionsProps['cves'];
+    isUpdate?: boolean;
```

Comment:
Nit: what do you think about `showScopeField` instead of `isUpdate`, to be in parallel with `showExpiryField`?

Reply:
Oh, I see what you mean. Since the form `ExceptionRequestForm` doesn't need to know whether you're creating a request or updating a request, we can remove that context. Yea, we can do that 👍🏼